modules/virt-about-nmstate.adoc

```diff
@@ -23,3 +23,8 @@ Node networking is monitored and updated by the following objects:
 * Bond
 
 * Ethernet
+
+[NOTE]
+====
+If your {product-title} cluster uses OVN-Kubernetes as the default Container Network Interface (CNI) provider, you cannot attach a Linux bridge or bonding to the default interface of a host because of a change in the host network topology of OVN-Kubernetes. As a workaround, you can use a secondary network interface connected to your host or switch to the OpenShift SDN default CNI provider. 
```

Comment:
Looks good. Would it make sense to link to the BZ? So we are reminded to drop this once it gets resolved?

https://bugzilla.redhat.com/show_bug.cgi?id=1885605

Reply:
@phoracek Thanks. As you know from your conversation with @aburdenthehand, a new will docs bug will be created that depends on the engineering bug. We will add the BZ link in the release notes.

Reply:
The line "As a workaround, you can use a secondary network interface connected to your host or switch to the OpenShift SDN default CNI provider. " gives me pause.
The term 'switch' in this context immediately makes me think of a network switch, which then reads "you can use a secondary network interface to connect to your host or (network) switch..." but I think we're actually saying " You can use a secondary network interface to connect to your host or *change* the cluster network provider to OpenShift SDN." 
Is that correct? @phoracek @sjhala-ccs

Reply:
@aburdenthehand @sjhala-ccs Sounds good to me. Another alternative might be adding a comma between "host" and "or switch":
As a workaround, you can use a secondary network interface connected to your host, or switch to the OpenShift SDN default CNI provider.

Or another option might be:
As a workaround, you can try any of the following:
 - Use a secondary network interface connected to your host
 - Switch to the OpenShift SDN default CNI provider.

Reply:
@aburdenthehand  "You can use a secondary network interface to connect to your host or change the cluster network provider to OpenShift SDN." That is what we are trying to say. Would adding a comma make it clearer?